examples/bukkit/src/main/java/dev/rollczi/example/bukkit/ExamplePlugin.java

```diff
@@ -25,29 +27,57 @@ public class ExamplePlugin extends JavaPlugin {
+        this.liteCommands = LiteCommandsBukkit.builder()
+            // configure bukkit platform
+            .settings(settings -> settings
+                .fallbackPrefix("my-plugin") // fallback prefix - used by bukkit to identify command
+                .nativePermissions(false) // enable/disable bukkit permissions system
+            )
+
+            // Commands
+            .commands(LiteCommandsAnnotations.of(
+                new ConvertCommand(),
+                new GameModeCommand(),
+                new KickCommand(),
+                new TeleportCommand()
+            ))
+            .commands(LiteCommandsProgrammatic.of(
+                new LiteCommand<CommandSender>("ban")
+                    .permissions("example.ban")
+                    .argument("player", Player.class)
+                    .onExecute(context -> {
+                        Player player = context.argument("player", Player.class);
+                        player.kickPlayer("You have been banned!");
+                    })
+            ))
+
+            // change default messages
+            .message(LiteBukkitMessages.LOCATION_INVALID_FORMAT, input -> "&cInvalid location format: &7" + input)
+
+            // Arguments @Arg
             .argument(GameMode.class, new GameModeArgument())
-            .argument(Player.class, new BukkitPlayerArgument<>(this.getServer(), "&cNie ma takiego gracza!"))
 
-            // Contextual Bind
-            .contextualBind(Player.class, new BukkitOnlyPlayerContextual<>("&cKomenda tylko dla gracza!"))
+            // Suggestions, if you want you can override default argument suggesters
+            .argumentSuggester(Integer.class, SuggestionResult.of("1", "2", "3"))
+            .argumentSuggester(String.class, JoinArgument.KEY, SuggestionResult.of("Simple suggestion", "Simple suggestion 2"))
 
-            // Commands
-            .command(TeleportCommand.class, KickCommand.class, ConvertCommand.class)
+            .message(LiteBukkitMessages.PLAYER_ONLY, "&cOnly player can execute this command!")
+            .message(LiteBukkitMessages.PLAYER_NOT_FOUND, input -> "&cPlayer &7" + input + " &cnot found!")
+
+            // Handlers for missing permissions and invalid usage
+            .missingPermission(new ExampleMissingPermissionsHandler())
+            .invalidUsage(new ExampleInvalidUsageHandler())
 
-            // Handlers
-            .invalidUsageHandler(new InvalidUsage())
-            .permissionHandler(new PermissionMessage())
+            // Schematic generator is used to generate schematic for command, for example when you run invalid command.
+            .schematicGenerator(SchematicFormat.angleBrackets())
 
-            .register();
+            .build();
     }
 
     @Override
     public void onDisable() {
-        this.liteCommands.getPlatform().unregisterAll();
+        // unregister all commands from bukkit
+        this.liteCommands.unregister();
```

Comment:
Check if it is null! any bad programmer using litecommands may have a bigger exception because of this!

```suggestion
        if (this.liteCommands != null) {
            this.liteCommands.unregister();
        }    
```